tests/rados/customer_scenarios.py

```diff
@@ -0,0 +1,451 @@
+    Creating various scenarios where the DB would be updated with new mappings and verify it DB is getting trimmed.
+
+    Verifies BZ:
+    https://bugzilla.redhat.com/show_bug.cgi?id=1905339
+    https://bugzilla.redhat.com/show_bug.cgi?id=1829646
+    https://bugzilla.redhat.com/show_bug.cgi?id=1972281
+    https://bugzilla.redhat.com/show_bug.cgi?id=1943357
+    https://bugzilla.redhat.com/show_bug.cgi?id=1766702
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): ceph cluster
+        node: Cephadm node where the commands need to be executed
+        kwargs: Any other KV pairs that need to be passed for testing
+    Returns: Exception hit for failure conditions.
+
+    """
+    mon_nodes = ceph_cluster.get_nodes(role="mon")
+    osd_nodes = ceph_cluster.get_nodes(role="osd")
+    client_node = ceph_cluster.get_nodes(role="client")[0]
+    installer_node = ceph_cluster.get_nodes(role="installer")[0]
+    daemon_info = run_ceph_command(node, action="daemon_info")
+    mon_daemons = [entry for entry in daemon_info if entry["daemon_type"] == "mon"]
+
+    # Duration for which we will sleep after the mon DB changes are made and mon would have begun trimming old mappings
+    mon_db_trim_wait_dur = 1200
+
+    # List to capture the mon db size throughout the duration of the test to check the variations in DB size
+    mon_db_size_list = list()
+    mon_db_size_list.append(get_mondb_size(mon_nodes[0], mon_daemons))
+
+    # Collecting first and last commits to osdmap
+    status = run_ceph_command(node, action="ceph_report")
+    init_commmits = {
+        "osdmap_first_committed": float(status["osdmap_first_committed"]),
+        "osdmap_last_committed": float(status["osdmap_last_committed"]),
+    }
+
+    # creating scenarios where the mon db would be updated with new info
+    change_config_for_slow_ops(node=node, action="set", **kwargs)
+    mon_db_size_list.append(get_mondb_size(mon_nodes[0], mon_daemons))
+
+    # Starting read and write on by creating a test pool .
+    pool_name = "test_pool_ops"
+    if not create_pool(node=node, pool_name=pool_name, crush_rule="stretch_rule"):
+        error = "failed to create pool to run IO"
+        raise TestCaseFailureException(error)
+    cmd = f"rados --no-log-to-stderr -b 1024 -p {pool_name} bench 400 write --no-cleanup &"
+    client_node.exec_command(sudo=True, cmd=cmd)
+
+    mon_db_size_list.append(get_mondb_size(mon_nodes[0], mon_daemons))
+
+    # deleting a previously created pool to increase OSD operations and map changes
+    # Pool created as part of suite set-up workflow.
+    detete_pool(node=node, pool="delete_pool")
+
+    # Proceeding to reboot 1 OSD from each host to trigger rebalance & Backfill
+    cluster_fsid = run_ceph_command(node, action="ceph fsid")["fsid"]
+    daemon_info = run_ceph_command(node, action="daemon_info")
+    osd_daemons = [entry for entry in daemon_info if entry["daemon_type"] == "osd"]
+    for val in osd_nodes:
```

Comment:
Instead of `val`, could we change it to `osd` or `node`? It's would become more readable.

Reply:
changed the name